src/views/Joseki/Joseki.tsx

```diff
@@ -259,6 +259,47 @@ class _Joseki extends React.Component<JosekiProps, JosekiState> {
 
         this.goban_div = document.createElement("div");
         this.goban_div.className = "Goban";
+
+        // arange translation of known joseki tags
+        // (these values come from the Joseki Tags table on the server)
+        try {
+            pgettext("This is a Joseki Tag", "Joseki: Done");
+            pgettext("This is a Joseki Tag", "Fuseki: Done");
+            pgettext("This is a Joseki Tag", "Current");
+            pgettext("This is a Joseki Tag", "Basic");
+            pgettext("This is a Joseki Tag", "Position is settled");
+            pgettext("This is a Joseki Tag", "Fighting pattern");
+            pgettext("This is a Joseki Tag", "Black gets the corner");
```

Comment:
I suppose reducing translator workload might be a reason to prefer the static solution:

```
function getTagText(tag_id: number) {
    switch (tag_id) {
        ...
    case 0x06:
        return interpolate(pgettext("Joseki Tag", "{{color}} gets {{result}}"), { color: _("Black"), result: pgettext("Joseki Tag Result", "the corner"));
    case 0x07:
        return interpolate(pgettext("Joseki Tag", "{{color}} gets {{result}}"), { color: _("Black"), result: pgettext("Joseki Tag Result", "the corner and top"));
        ...
}
```

That only cuts the translations roughly in half, but given that OGS supports 64 languages and there are 34 Joseki tags, I thought it was worth mentioning.

Reply:
On the other hand, interpolation adds complexity for translators too, so perhaps the costs outweight the benefits...

Reply:
Single words like color can work sometimes, more complicated multi word pastes like that though usually don't translate so well due to grammatical differences

Reply:
Where does this leave us?

Reply:
We're good, was just on my phone when typing that so wasn't the time to merge - doing so now, thanks for the improvement!

Reply:
Woo hoo and phew.

Now I just have to do the "Categories" and OJE should be just about fully done (Also Empty Board and Tenuki button).

Reply:
No drama at alll - it's great to have review & options :)     Alex's app does support OJE, so that was one reason (in addition to inertia) why I stuck with what we have :D